### PR TITLE
enhancement

### PR DESCRIPTION
**Pull request by 0xIslamTaha**
code enhancement

### CODE REVIEW

File: install/tests/base_test.py [hunk 1]
```diff
@@ -1,80 +1,79 @@
...
 
-    def check_js_container_installtion(self):
-        self.info(" Check that js container exist ,should succeed")
+    @staticmethod
+    def is_js_container_installed():
+        BaseTest.info(" Check that js container exist ,should succeed")
         command = "/tmp/jsx container-kosmos"
-        output, error = self.os_command(command)
-        if error:
-            output, error = self.jumpscale_installtion("container-install", "-n {}".format(self.js_container))
```
Comment (Dinaamagdy) [inline install/tests/base_test.py:74]: need jumpscale installtion if not exist here

File: install/tests/test_installtion.py [hunk 2]
```diff
@@ -1,49 +1,42 @@
...
 
 
-class Test_instaltion(BaseTest):
+class TestInstallationInDocker(BaseTest):
     def setUp(self):
-        pass
+        self.CONTAINER_NAME = str(uuid.uuid4()).replace("-", "")[:10]
+        self.info(
+            "Install container jumpscale from {} branch in {} os type".format(self.get_js_branch(), self.get_os_type()))
+        output, error = self.jumpscale_installation("container-install", "-n {}".format(self.CONTAINER_NAME))
```
Comment (Dinaamagdy) [inline install/tests/test_installtion.py:10]: why we install jumpscale every testcase ? we just need it one time

File: install/tests/base_test.py (hunk 1 above)
Comment (0xIslamTaha) [reply install/tests/base_test.py:74]: according to the new setup and teardown methods, there is no test case calling this method anymore.

File: install/tests/test_installtion.py (hunk 2 above)
Comment (0xIslamTaha) [reply install/tests/test_installtion.py:10]: actually, we are testing the installation itself with a specific action that is why we need to install/remove in setup/teardown. In general, the test case should be independent of each other. So, if one of the test cases ruin the jsx installation, we will be sure it won't affect the other test cases.

**Pull request merged**